capa/render/result_document.py

```diff
@@ -501,9 +505,20 @@ class MaecMetadata(FrozenModel):
     model_config = ConfigDict(frozen=True, populate_by_name=True)
 
 
+@lru_cache(maxsize=None)
+def load_rules_prevalence() -> Dict[str, str]:
+    CD = capa.main.get_default_root()
+    file = CD / "assets" / "rules_prevalence_data" / "rules_prevalence.json.gz"
+    if not file.exists():
+        return {}
+    with gzip.open(file, "rb") as gzfile:
+        return json.loads(gzfile.read().decode("utf-8"))
```

Comment:
while we're at it, is it worth defining a pydantic data model for the DB file/format?

Reply:
looks like the format is `dict[rule name, prevalence]` which will be hard to represent in pydantic, unless we enumerate all the rule names as potential values. i think the type hint above is a good start. still, adding some comments here showing a snippet of the file would be valuable.